src/bin/zmimapdctl

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+# 
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Server
+# Copyright (C) 2017 Synacor, Inc.
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program.
+# If not, see <https://www.gnu.org/licenses/>.
+# ***** END LICENSE BLOCK *****
+# 
+
+source `dirname $0`/zmshutil || exit 1
+zmsetvars
+
+if [ ! -d ${zimbra_java_home} ]; then
+    (>&2 echo "Did not find java at '${jar_prefix}'")
```

Comment:
I know we don't have a shell style guide, but if we did I'd probably vote for following something like https://google.github.io/styleguide/shell.xml

For this particular line, I'd recommend removing the parenthesis and placing >&2 a the end of the line.